pkg/cmd/eventlistener/testdata/TestEventListenerDescribe_WithCELInterceptors.golden

```diff
@@ -0,0 +1,23 @@
+Name:        el1
+Namespace:   ns
+
+Event Listener Triggers
+
+EventListenerTrigger
+
+ NAME
+ foo-trig
+ BINDING NAME    KIND             APIVERSION
+ tb1             TriggerBinding   
+ tb2                              v1alpha1
+ tb3             TriggerBinding   v1alpha1
+ TEMPLATE NAME   APIVERSION
+ tt1             v1alpha1
+ INTERCEPTORS
+- cel:
+    filter: body.value == 'test'
+    overlays:
+    - expression: '''testing'''
+      key: value
+
```

Comment:
Can we find a way to avoid the extra line break with interceptors?

Reply:
I have removed extra line